sale_commission/__openerp__.py

```diff
@@ -1,54 +1,52 @@
 # -*- coding: utf-8 -*-
 # © 2011 Pexego Sistemas Informáticos (<http://www.pexego.es>)
 # © 2015 Avanzosc (<http://www.avanzosc.es>)
-# © 2015 Pedro M. Baeza (<http://www.serviciosbaeza.com>)
+# © 2015-2016 Pedro M. Baeza (<http://www.serviciosbaeza.com>)
+# © 2015-2016 Oihane Crucelaegui
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 {
     'name': 'Sales commissions',
-    'version': '8.0.2.3.0',
+    'version': '9.0.1.0.0',
     'author': 'Pexego, '
               'Savoire-faire linux, '
               'Avanzosc, '
               'Abstract, '
               'Serv. Tecnol. Avanzados - Pedro M. Baeza, '
               'Odoo Community Association (OCA)',
-    "category": "Sales Management",
+    'category': 'Sales Management',
     'license': 'AGPL-3',
     'depends': [
         'base',
         'account',
         'product',
-        'sale'
+        'sale',
     ],
     'contributors': [
```

Comment:
There is a field in ir_module_module that is filed with this data, that's why I include it.